storybook/src/docs/developer-guide/release-policy.docs.mdx

```diff
@@ -0,0 +1,73 @@
+{/* @license CC0-1.0 */}
+
+import { Meta } from "@storybook/addon-docs/blocks";
+
+<Meta title="Docs/Developer Guide/Release Policy" />
+
+# Release Policy
+
+We aim to publish new releases on a monthly basis.
+Breaking changes will occur at most once every 3 months.
+Change logs, release notes, and our sprint reviews help you with upgrades.
+
+## Packages
+
+We publish the code libraries of our design system as **5 separate npm packages**:
+
+1. [Assets](https://www.npmjs.com/package/@amsterdam/design-system-assets): our fonts, icons, logos and application icons.
+2. [CSS](https://www.npmjs.com/package/@amsterdam/design-system-css): applies our design to all components and utilities.
+3. [React](https://www.npmjs.com/package/@amsterdam/design-system-react): implements our component library in React.
+4. [React Icons](https://www.npmjs.com/package/@amsterdam/design-system-react-icons): all of our icons as React components.
+5. [Tokens](https://www.npmjs.com/package/@amsterdam/design-system-tokens): CSS variables that express our design choices.
+
+## Version numbering
+
+Each package has its **own version number**.
+The design system as a whole **does not have** a single overarching version.
+Because of cross-dependencies, the 3 CSS and React packages will often move to a new major version together.
```

Comment:
Do you mean the CSS, React and tokens packages? React icons will probably not move to a new major version as often.